### PR TITLE
Wip first draft of a prune command.

### PR DESCRIPTION
**Pull request by mikeSimonson**
What do you think of it now.

TODO:
 - tests

### CODE REVIEW

File: src/Bowerphp/Command/PruneCommand.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?php
+
+/*
+ * This file is part of Bowerphp.
+ *
+ * (c) Massimiliano Arione <massimiliano.arione@bee-lab.net>
```
Comment (garak) [inline src/Bowerphp/Command/PruneCommand.php]: Feel free to add your name/email here

**Comment by garak** [comment]
Well, it seems too much logic in Command to me.
You should think to a Command like if it were a Controller: so it should be as thin as possible, while all logic should be pushed in Model, if possible.

**Comment by mikeSimonson** [comment]
I didn't moved it there yet because the distinction between the responsibilities of a package, a repository and the bowerphp file.

Not sure where to put what.

**Comment by garak** [comment]
If you are in doubt, put it in ``Bowerphp`` class.
Also, try to avoid private methods in Command, if possible.
Also, direct access to ``bower.json`` file from a Command is a big big bad smell.

**Comment by garak** [comment]
Hi @mikeSimonson, any news about this PR?

**Comment by mikeSimonson** [comment]
Hi @garak 
It will be done by next week.

**Comment by garak** [comment]
Sorry Mike, this PR is now too old. Feel free to open a new one.
Thank you.

**Pull request closed without merging**